libraries/AP_NavEKF2/AP_NavEKF2_core.h

```diff
@@ -1151,7 +1151,7 @@ class NavEKF2_core
     } mag_state;
 
     // string representing last reason for prearm failure
-    char prearm_fail_string[40];
+    char prearm_fail_string[40]{"not enabled"};
```

Comment:
Why? It doesn't make sense, if the core exists than it's enabled...

Reply:
really?